.prettierignore.include

```diff
@@ -12,3 +12,6 @@ testdata/
 # Everything in site/ is formatted by Biome. For the rest of the repo though, we
 # need broader language support.
 site/
+
+# Prettier messes up markdown alignment in the documentation
+**/docs/*.md
```

Comment:
is it needed here?

Reply:
yeah - goal for now is to keep prettier from messing up the markdown spacing